code/modules/paperwork/paper.dm

```diff
@@ -82,6 +82,9 @@
 		name = "paper[(n_name ? text("- '[n_name]'") : null)]"
 	add_fingerprint(usr)
 
+/obj/item/weapon/paper/suicide_act(mob/user)
+	user.visible_message("<span class='suicide'>[user] gives \himself a thousand tiny paper cuts! It looks like \he's trying to commit suicide..</span>")

```

Comment:
Yeah it's in pen and newscasters, don't credit me with it, it's an ancient joke to replace commit seppuku with commit sudoku.